ultravox/training/evaluation.py

```diff
@@ -113,13 +114,17 @@ def evaluate(
     num_procs: int = 8,
     max_new_tokens: Optional[int] = None,
     temperature: Optional[float] = None,
-    verbose: bool = False,
+    log_dir: Optional[str] = None,
```

Comment:
should this be enabled by default?

Reply:
I don't understand the question. Are you suggesting it should be and it's not. Or are you suggesting it shouldn't be enabled and it is?

Reply:
I assume that when evaluate() is called, by default the caller will pass in a log_dir, correct? If that's the case, please ignore the comment. I want to make sure that by default logging of eval results are enabled.

Reply:
I saw it was set properly in trainer.py. This is all good.